Finish redesign

archive64 opened the pull request.

Commit:
commit build files

Commit:
redesign: show toolbar icons ltr always

The current redesign stylesheet doesn't display
nicely if you set dir="rtl". For example, this
happens if you set Firefox's preferred language
to Arabic.

For now, just default to "ltr" all of the time.
We'll look into re-enabling localization features
in a future commit.

Commit:
commit build files

Commit:
redesign: prevent icon scrollbars on IE/Edge

Commit:
redesign: use -webkit flex properties

This didn't work on iPad Air 2 iOS8. Add
some older vendor prefixes and property names.

Commit:
commit build files

solflare commented on build/generic/web/viewer.html:
```diff
@@ -51,22 +53,22 @@
+<?= $listIcon ?> 
+            <span data-l10n-id="outline_label">Document Outline</span>
+         </button>
+         <button id="viewAttachments" class="toolbarButton group hidden" title="Show Attachments" tabindex="4" data-l10n-id="attachments">
```
Do we still want the `group` class here?

archive64 replied:
It doesn't really matter since it's hidden anyway.

Also, you can skip the commits that say "commit build files." They contain changes that have already been reviewed in other pull requests.

solflare commented on build/generic/web/viewer.html:
```diff
@@ -184,30 +191,36 @@
                 </a>
 
-                <button id="secondaryToolbarToggle" class="toolbarButton visibleLargeView" title="Tools" tabindex="37" data-l10n-id="tools">
+                <button id="secondaryToolbarToggle" class="toolbarButton hidden" title="Tools" tabindex="37" data-l10n-id="tools">
```
We don't want this visible ever?

archive64 replied:
Yep! That's what the designers requested.

Commit:
redesign: fix bad scrollbar look in Edge

The autohiding scrollbar made it so you couldn't
drag it.

The top button looked weird when hidden 5px under
the toolbar.

Commit:
commit build files
